product-service/handlers/product-item.ts

```diff
@@ -6,38 +6,275 @@ import {
 import 'source-map-support/register';
 import { throwError } from '../helpers/error.helper';
 import { ErrorsEnum } from '../types/errors.enum';
-import data from '../fake-data/MOCK_DATA.json';
-import { IProductItem } from '../types/product-item.interface';
 import {
     formattedErrorResponse,
     formattedSuccessResponse,
 } from '../helpers/response.helper';
+import { dbOptions } from '../helpers/db.helper';
+import * as pg from 'pg';
+import { isUUIDv4 } from '../helpers/request.helper';
 
 export const getProductsById: APIGatewayProxyHandler = async (
     event: APIGatewayProxyEventBase<APIGatewayEventDefaultAuthorizerContext>
 ) => {
+    console.log('📝 getProductsById: ', event);
```

Comment:
[ minor ] There are 4 different handlers in one file. I would consider some refactoring to put each handler to its own file.

Reply:
alright, thx, I thought all similar functionality goes into one handler